CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_zuora v0.2.1
```

Comment:
I am not entirely opposed to this being a patch upgrade rather than a breaking change. However, would you be able to share your reasoning as to why this is a patch and not a breaking change?

Reply:
@fivetran-joemarkiewicz My thinking was that it was a patch since it was correcting test failures, but on the other hand I did introduce aggregations in `zuora__line_item_history` which a user might not expect. I can make this breaking if that's safer?

Reply:
Based on our discussion today, I will keep this as a patch!